code/game/objects/items/weapons/blades.dm

```diff
@@ -71,7 +71,7 @@
 	return ..()
 
 /obj/item/weapon/combat_knife
-	name = "\improper M5 'Night Raider' survival knife"
+	name = "\improper M5 \"Night Raider\" survival knife"
```

Comment:
why

Reply:
Because there are no quotation marks instead of apostrophe? Also, I need the slashes for the quotation marks because it'll break without them.

Reply:
Anyways, reverted now.